DOCS-2056: Additional vision config card changes (#3165)
viamrobotics/docs: docs/tutorials/projects/filtered-camera.md

type: "docs"
description: "Use the filtered-camera module to selectively capture images."
images: ["/tutorials/filtered-camera-module/viam-figure-preview.png"]

> imageAlt: "The wooden Viam figure being detected on a camera stream"

Reviewer: ```suggestion
imageAlt: "A wooden Viam figure being detected on a camera stream"
```

For more information, see [Configure an `mlmodel` detector](/services/vision/mlmodel/).

## Test your ML model

Before filtering your data, test that your ML model is working as expected with your camera.

You can test your detector by clicking on the **Test** area of the vision service's configuration panel or from the [**CONTROL** tab](/fleet/control/).

The camera stream will show detections with bounding boxes around the detections, so try placing an object your ML model can recognize in front of the camera.

{{< imgproc src="/tutorials/filtered-camera-module/transform-camera-overlay.png" alt="Detection of a viam figure with a confidence score of 0.97" resize="800x" >}}

> When satisfied that your ML model is working well, continue.
> Alternatively,if the vision service is not matching reliably, you will need to adjust your model.

Reviewer: ```suggestion
When satisfied that your ML model is working well, continue to the next step.
If the vision service is not matching reliably, you will need to adjust your model.
```
"continue" on its own sounds awk to me

If you trained your model, consider adding and labelling more images in your dataset.

> Ideally, you want your ML model to be able to identify objects with a high level of confidence, which usually is dependent on a robust source dataset.

Reviewer: ```suggestion
Ideally, you want your ML model to be able to identify objects with a high level of confidence, which is usually dependent on a robust source dataset.
```
